src/main/twirl/views/manage/previewOppSection.scala.html

```diff
@@ -0,0 +1,36 @@
+@(
+opp: Opportunity,
+sectionNumber: Int
+)
+
+@formatId(id: Long) = @{
+f"RIFS$id%05d"
+}
+
+@currentSection = @{
+opp.description.find(_.sectionNumber == sectionNumber)
+}
+
+@main(s"${opp.title} - RIFS", backLink=Some(BackLink("Opportunity template",controllers.manage.routes.OpportunityController.showOverviewPage(opp.id).url)), displayUserName=Some("Portfolio Peter")) {
+<div class="grid-row">
```

Comment:
It would probably be better to lose these two heading divs, (grid-row and column-full) and just wrap this in a `<section>` instead.